### DISCUSSION TITLE
Downloads fix ideas

### DISCUSSION THREAD

**Discussion by Hello-dot-Jpg** [nk64/GT4286#4] [Ideas]
I suspect the original intention with the downloads folder was to add the games to the database then move the roms to the correct roms folder and remove them from downloads I'm not sure why they didn't implement it this way 

it would be nice if we could add the ability to delete installed roms if we can get the above functionality to work removing the rom and the SQLite db row

**Comment by nk64** [comment]
I don't think that the download functionality was ever designed to move the files to the roms folder. There is an int column in the game table (in game.db) called ```dwonloadid``` (sic) for whether the rom is in the download folder or not, so from their perspective a factory reset is as easy as deleting all the files in the /download directory and then running the SQL ```delete from game where dwonloadid > 0``` (or something like that).

**Reply by Hello-dot-Jpg** [reply]
I think they also use that column for the Clear and Delete entry buttons in the Menu on the download's page
Similarly, they use fav for Favorites and his for History and clear the value in the row or the table depending on what was selected

I haven't tried the factory reset yet, but seeing as the text is cut off, I am guessing they never did either :D
